packages/components/pie-button/tsconfig.json

```diff
@@ -1,8 +1,11 @@
 {
-  "extends": "../../../configs/pie-components-config/tsconfig.json",
+  "extends": "@justeattakeaway/pie-components-config/tsconfig.json",
   "compilerOptions": {
     "baseUrl": ".",
     "rootDir": ".",
+    "paths": {
```

Comment:
If we're defining this `paths` setting in most of our components/tools already, why not leave it in and override it when wee don't want it set to this?

Reply:
See this comment :) - https://github.com/justeattakeaway/pie/pull/1159#discussion_r1447161107

Reply:
Caught up with @ashleynolan about this, and we'll be leaving the `paths` in the component TSConfig 👍🏼